main.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- create a navbar as a header  -->
+    <header>
+        <nav class="nav">
+            <ul class="ul ul1">
+                <li class="navli nav-li1">My logo</li>
+                <li class="navli nav-li2">Portfolio</li>
+                <li class="navli nav-li3">About</li>
+                <li class="navli nav-li4">Contact</li>
+            </ul>
+        </nav>
+    </header>
+
+        <section class="main-section"><!--container with background image-->
+            <section class="text-section"><!--container for text and flexbox for text-->
+            <!-- text of the main page -->
+                <h1 class="box h1">I’m Isaac Glad to see you!</h1>
+                <p class="box p1">I’m a software developer! I can help you build a product , feature or website Look through some of my
+                        work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="box p2">LET’S CONNECT</p>
+                <ul class="box ul ul2">
+                    <li><img src="img\twitter@2x.png" alt="twitter"></li>
+                    <li><img src="img\Linkedin icon@2x.png" alt="Linked In"></li>
+                    <li><img src="img\medium@2x.png" alt="Medium"></li>
+                    <li><img src="img\github@2x.png" alt="Github"></li>
+                    <li><img src="img\angellist@2x.png" alt="Angelist"></li>
+                </ul>
```

Comment:
- Please, kindly wrap the images in an anchor tag so you can add the links to your many social media platforms of you to ensure people can connect with you from your portfolio and meet the project requirement.